challenges-22-plus.yml

```
@@ -0,0 +1,192 @@
+lazy_frame_full_frame:
+  question: |
+    My animal rescue website has a page with very useful contact numbers that
```

```suggestion
    My animal rescue website has a page with very useful contact numbers.
```

(and remove the rest of the sentence)